test/integration.js

```diff
@@ -18,12 +18,12 @@ describe('The Paysage server', function () {
     server.close(done);
   });
 
-  it('lists existing playgrounds', function () {
-    world.getOrCreatePlayground('here');
-    return request
-      .get('/list')
+  it('lists playgrounds', function (done) {
+    var herePlayground = world.getOrCreatePlayground('here');
+    herePlayground.getOrCreateCodeObject('bill');
+    request.get('/list')
       .expect(200)
-      .expect(/here/);
+      .expect(/here/, done);
```

Comment:
What made you think the original test was not doing anything (in the old test, I tried changing `/here/` to /not-here/ and it broke correctly)? It's legal in mocha to return a promise instead of using the `done` callback.

Reply:
Effectively. I removed the return for an other reason and forgot to put it back…